pkg/ast/pipesearch/searchHandler.go

```diff
@@ -204,9 +205,24 @@ func ParseSearchBody(jsonSource map[string]interface{}, nowTs uint64) (string, u
 			scrollFrom = 0
 		}
 	}
+
+	includeNullsVal, ok := jsonSource["includeNulls"]
+	if !ok {
+		includeNulls = false
+	} else {
+		switch val := includeNullsVal.(type) {
+		case bool:
+			includeNulls = val
+		case string:
+			includeNulls = val == "true"
+		default:
+			includeNulls = false
+		}
```

Comment:
You should also have a log message here, saying we got an unexpected type, and specifying what type it is